include/device.h

```diff
@@ -85,6 +85,25 @@ extern "C" {
 	DEVICE_AND_API_INIT(dev_name, drv_name, init_fn,\
 	data, cfg_info, level, prio, NULL)
 
+/**
+ * @def DEVICE_INIT
```

Comment:
This should be `@def DEVICE_INIT_ALL_INSTANCES`

Reply:
oh yes thanks!